invenio_communities/config.py

```diff
@@ -28,7 +28,7 @@
 
 """Communities ui endpoints."""
 
-COMMUNITIES_ALLOW_GROUP_INVITES = True
+COMMUNITIES_ENABLED = True
```

Comment:
I am confused on this one. I thought we are removing `COMMUNITIES_ENABLED` ? ping @ntarocco

Reply:
As far as I understood we will have communities enabled (not allowed to disabled), however groups can be enabled/disabled.

```suggestion
COMMUNITIES_GROUPS_ENABLED = True/False
```